cmd/agent.go

```diff
@@ -28,10 +29,28 @@ import (
 	"github.com/open-policy-agent/opa/rego"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
-
-	"github.com/compliance-framework/configuration-service/sdk"
 )
 
+// TODO: Where should this live?
+type resultHelper struct {
+	results []*proto.AssessmentResult
+}
+
+func NewResultsHelper() *resultHelper {
+	return &resultHelper{
+		results: []*proto.AssessmentResult{},
+	}
+}
+
+func (h *resultHelper) CreateResult(assessmentResult *proto.AssessmentResult) error {
+	h.results = append(h.results, assessmentResult)
+	return nil
+}
```

Comment:
Instead of appending a list and sending after Eval, this method should be doing the sending. For now, we'll just send straight to the API. Later we will amend this to support disconnected environments. For now, straight to the API.